cypress/e2e/DoenetML/tagSpecific/regularPolygon2.cy.js

```diff
@@ -12,62 +12,62 @@ describe("Regular Polygon Tag Tests 2", function () {
   it("specify radius for square", () => {
     setupRegularPolygonScene({
       attributes: {
-        nVertices: "4",
+        numVertices: "4",
         radius: "7",
         center: "(-6,-2)",
       },
     });
 
     runRegularPolygonTests({
-      nVertices: 4,
+      numVertices: 4,
       vertex1: [-6 + 7, -2],
       center: [-6, -2],
-      conservedWhenChangeNvertices: "circumradius",
+      conservedWhenChangenumVertices: "circumradius",
       abbreviated: true,
     });
   });
 
   it("specify center for pentagon", () => {
     setupRegularPolygonScene({
       attributes: {
-        nVertices: "5",
+        numVertices: "5",
         center: "(-5,-3)",
       },
     });
 
     runRegularPolygonTests({
-      nVertices: 5,
+      numVertices: 5,
       vertex1: [-5 + 1, -3],
       center: [-5, -3],
-      conservedWhenChangeNvertices: "circumradius",
+      conservedWhenChangenumVertices: "circumradius",
     });
   });
 
   it("specify one vertex for square", () => {
     setupRegularPolygonScene({
       attributes: {
-        nVertices: 4,
+        numVertices: 4,
         vertices: "(2,-5)",
       },
     });
 
     runRegularPolygonTests({
-      nVertices: 4,
+      numVertices: 4,
       vertex1: [2, -5],
       center: [1, -5],
-      conservedWhenChangeNvertices: "circumradius",
+      conservedWhenChangenumVertices: "circumradius",
```

Comment:
I know it ends up case insensitive when parsing, but wanted to check if you were interested in getting the camel case right in the JS code while we are doing these replacements.

Reply:
Might as well.  That looks weird.  I'll fix that one.